docs/basics/Epics.md

```diff
@@ -25,32 +26,29 @@ If you let an incoming action pass through, it will create an infinite loop:
 
 ```js
 // DO NOT DO THIS
-const actionEpic = (action$) => action$; // creates infinite loop
+const actionEpic = action$ => action$; // creates infinite loop
 ```
 
-> The pattern of handling side effects this way is similar to the "*process manager*" pattern, sometimes called a ["*saga*"](https://msdn.microsoft.com/en-us/library/jj591569.aspx), but the original definition of [saga is not truly applicable](http://kellabyte.com/2012/05/30/clarifying-the-saga-pattern/). If you're familiar with [redux-saga](https://redux-saga.github.io/redux-saga/), redux-observable is very similar. But because it uses RxJS it is much more declarative and you utilize and expand your existing RxJS abilities.
-
+> The pattern of handling side effects this way is similar to the "_process manager_" pattern, sometimes called a ["_saga_"](https://msdn.microsoft.com/en-us/library/jj591569.aspx), but the original definition of [saga is not truly applicable](http://kellabyte.com/2012/05/30/clarifying-the-saga-pattern/). If you're familiar with [redux-saga](https://redux-saga.github.io/redux-saga/), redux-observable is very similar. But because it uses RxJS it is much more declarative and you utilize and expand your existing RxJS abilities.
 
 ## A Basic Example
 
 > **IMPORTANT:** redux-observable does not add any of the RxJS operators to the `Observable.prototype` so you will need to import the ones you use or import all of them in your entry file.
-> 
+>
 > Because there are many ways to add them, our examples will not include any imports. If you want to add every operator, put `import 'rxjs';` in your entry `index.js`. [Learn more](http://redux-observable.js.org/docs/Troubleshooting.html#rxjs-operators-are-missing-eg-typeerror-actionoftypeswitchmap-is-not-a-function).
 
-
 Let's start with a simple Epic example:
 
 ```js
 const pingEpic = action$ =>
-  action$.filter(action => action.type === 'PING')
-    .mapTo({ type: 'PONG' });
-    
+  action$.filter(action => action.type === 'PING').mapTo({ type: 'PONG' });
```

Comment:
I'd like app epics to be consistent even if they don't have a lot of operators applied. The other cases where there were 3+ prettier decided to put them all on newlines like

```js
action$
  .filter(action => action.type === 'PING')
  .mapTo({ type: 'PONG' });
```

Not sure if this is easy to do.